ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/DecoderOptions.java

```diff
@@ -14,6 +14,12 @@
  */
 public interface DecoderOptions extends SinkOptions, PipelineOptions {
 
+  @Description("Static IP associated with load balancer; will be ignored for IP lookups")
+  @Default.String("35.227.207.240")
+  String getProxyIpAddress();
+
+  void setProxyIpAddress(String value);
+
```

Comment:
For simplicity, I've chosen to have this new config option be a single-entry string, and have the default set to the expected prod value.

This could be turned into a comma-separated list in the future.